flutter-for-android.md

```diff
@@ -2019,3 +2019,59 @@ class SampleApp extends StatelessWidget {
   }
 }
 {% endprettify %}
+
+
+# Databases
+## How do I access Shared Preferences in Flutter?
+In Android, you can store a small collection of key-value pairs by using
+the SharedPreferences API.
+
+In Flutter, you can access this functionality by using the
+Shared Preferences plugin [Shared_Preferences](https://pub.dartlang.org/packages/shared_preferences)
+
+This plugin wraps the functionality of both Shared Preferences and 
+NSUserDefaults (the iOS equivalent).
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+void main() {
+  runApp(new MaterialApp(
+    home: new Scaffold(
+      body: new Center(
+      child: new RaisedButton(
+        onPressed: _incrementCounter,
+        child: new Text('Increment Counter'),
+        ),
+      ),
+    ),
+  ));
```

Comment:
missing trailing comma here

Reply:
Let me know if this is good, I ran the formatter on it in intellij!